core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -23,23 +23,98 @@ package cats
 package syntax
 
 trait ApplySyntax extends TupleSemigroupalSyntax {
-  implicit final def catsSyntaxApply[F[_], A](fa: F[A])(implicit F: Apply[F]): Apply.Ops[F, A] =
+  @deprecated("Kept for binary compatibility", "2.10.0")
+  final def catsSyntaxApply[F[_], A](fa: F[A], F: Apply[F]): Apply.Ops[F, A] =
     new Apply.Ops[F, A] {
       type TypeClassType = Apply[F]
 
       val self = fa
       val typeClassInstance = F
     }
 
+  implicit final def catsSyntaxApplyFABOps[F[_], A, B](fab: F[A => B]): ApplyFABOps[F, A, B] =
+    new ApplyFABOps[F, A, B](fab)
+
+  implicit final def catsSyntaxApplyFABCOps[F[_], A, B, C](ff: F[(A, B) => C]): ApplyFABCOps[F, A, B, C] =
+    new ApplyFABCOps[F, A, B, C](ff)
+
   implicit final def catsSyntaxApplyOps[F[_], A](fa: F[A]): ApplyOps[F, A] =
     new ApplyOps(fa)
+
 }
 
 private[syntax] trait ApplySyntaxBinCompat0 {
   implicit final def catsSyntaxIfApplyOps[F[_]](fa: F[Boolean]): IfApplyOps[F] =
     new IfApplyOps[F](fa)
 }
 
+final class ApplyFABOps[F[_], A, B](private val fab: F[A => B]) extends AnyVal with Serializable {
```

Comment:
See https://github.com/typelevel/cats/pull/4477#discussion_r1282062073. It shouldn't be needed if the syntax classes are not being allocated, which is the hope 😅 

I don't know enough of the history but at some point it seems that people did care that these were serializable /shrug

Reply:
> They are not supposed to be saved nor transferred anywhere, are they?

I have no idea. I think typically serializability is important when you are dealing with e.g. Spark. Maybe in some (pathological) situations these classes can get caught up in the serialization graph somehow ...

Reply:
🤔 yeah, not sure either. I would probably suggest to remove `Serializable` for now. Just because it is always easier to add something later rather than remove it whenever. We are not introducing any binary incompatibility anyway, so if Spark guys or anyone else start complaining, then we can easily issue a hotfix release. But at least we will know for sure in that case :) Although I'd guess there will be no complains about it.

Otherwise, if we decide to go the opposite way, then we should add `Serializable` all the remaining syntax extension classes, shouldn't we?

Reply:
I'm fine with removing it. Anyway I think all Spark users are stuck on Cats 2.0.0-M5 or something 😛

Reply:
Sorry, I carelessly made ApplyFABOps Serializable🤦‍♂️ 
Thanks for pointing that out.